app/src/main/java/com/example/scoutconcordia/MapInfoClasses/ShuttleInfo.java

```diff
@@ -565,12 +566,12 @@ public double getEstimatedRouteTimeFromLoyola() {
     }
 
 
+    @RequiresApi(api = Build.VERSION_CODES.KITKAT)
     private String[] retrieveFridayLoyola() {
 
         ArrayList<String> shuttleArrayList = new ArrayList<>();
 
-        try {
-            InputStream is = context.getResources().openRawResource(raw.schedule_friday_loyola);
+        try(InputStream is = context.getResources().openRawResource(raw.schedule_friday_loyola)) {
```

Comment:
What does this change do and how does it resolve the SonarQube issue?

Same thing for all the other try statements that were modified.

Reply:
Great question, I had no idea what this was before I looked into it either!

SonarQube gave us this bug:
![image](https://user-images.githubusercontent.com/26283932/78463744-91292000-76ae-11ea-93cb-74c1cf7d550c.png)

@vicmanea and I both tried closing the streams in a finally block, however we kept getting errors. Therefore, I decided to lookup what try-with-resources did. Essentially, it's a try statement that declares resources. So, what this line is doing is simply declaring the resources that we will be using as input. The try-with-resources statements ensure that each resource is closed at the end of the statement so there is no leak if we do not close it since it's done automatically.